Makefile

```diff
@@ -159,6 +159,15 @@ else
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(GO_FLAGS) ./apis/... ./pkg/... ./test/yq/... ./controllers/... -covermode=atomic -coverprofile coverage.out
 endif
 
+.PHONY: vector-test
+vector-test: ## Run vector tests
```

Comment:
I didn't want it to depend on the vector-install if some person (well, maybe just me?) has it installed through brew or other means. Also, since it was broken on some platforms like MacOS (the fix isn't merged yet), it would make running the tests difficult.